msteams-platform/bots/calls-and-meetings/calls-meetings-bots-overview.md

```diff
@@ -22,6 +22,28 @@ The Teams [app manifest](../../resources/schema/manifest-schema.md#bots) with tw
+| Mid-call control | - Turn the video on or off<br>- Mute or unmute the microphone<br>- Switch between cameras<br>- Put the call on hold or resume locally<br>- Active speaker<br>- Choose speaker for calls<br>- Choose microphone for calls<br>- Show the state of a participant, such as idle, early media, connecting, connected, on hold, in lobby, and disconnected<br>- Show the state of a call, such as early media, incoming, Connecting, ringing, connected, on hold, disconnecting, or disconnected.<br>- Show if a participant is on mute<br>- Show the reason why a participant left a call |
+| Screen sharing | - Share the entire screen from within the app <br>- Share a specific application (from the list of running apps)<br>- Share a web browser tab from the list of open tabs<br>- Share system audio during screen sharing<br>- Participant can view remote screen share |
+| Roster | - List participants <br>- Remove a participant |
+| PSTN | - Place a one-to-one call with a PSTN participant <br>- Place a group call with PSTN participants<br>- Upgrade a one-to-one call with a PSTN participant into a group call<br>- Dial-out from a group call as a PSTN participant<br>- Support for early media |
+| General | Test your mic, speaker, and camera with an audio testing service (available by calling 8:echo123) |
+| Device management | - Ask for permission to use audio or audio and video <br>- Get camera list<br>- Set camera<br>- Get selected camera<br>- Get microphone list<br>- Set microphone<br>- Get selected microphone<br>- Get speakers list<br>- Set speaker<br>- Get selected speaker |
+| Video rendering | - Render single video in many places (local camera or remote stream) <br>- Set or update scaling mode<br>- Render remote video stream |
+
+
+## Set up firewalls
+
+Setting up firewalls is crucial to protect your network and system from unauthorized access and malicious attacks. Firewalls act as a barrier, controlling incoming and outgoing traffic based on security rules, preventing hackers from accessing your system, blocking malicious traffic, and hiding internal IP addresses. They provide an additional layer of security, preventing attacks like hacking, malware, and DoS attacks, and protect sensitive data from breaches. For more information, see https://cdn.graph.office.net/prod/pdf/teams-to-bot-communication-final.pdf.
+In the context of Microsoft Teams, setting up firewalls for calling and meeting bots is particularly important to protect sensitive communication data and prevent unauthorized access. Firewalls can help prevent toll fraud and denial-of-service (DoS) attacks on your bots, minimizing the risk of financial losses and disruption of critical communication services.
+For more information, see https://cdn.graph.office.net/prod/pdf/teams-to-bot-communication-final.pdf.
```

Comment:
```suggestion
```

Reply:
@v-shazilms, the doc PM has confirmed that we cannot use the provided PDF. Could you please suggest an alternative link, preferably from Microsoft Learn documentation?